addons/mail/static/src/models/thread.js

```diff
@@ -2240,6 +2227,7 @@ registerModel({
             inverse: 'channel',
             isCausal: true,
         }),
+        seen_message_id: attr(),
```

Comment:
`seen_message_id` is wrong as a field on thread, it's a field on channel member.
If it's on thread, name should still contain notion of "current partner/user".